mmv1/third_party/terraform/services/sql/resource_sql_database_instance_test.go.tmpl

```diff
@@ -4843,6 +4890,49 @@ func verifyPscAutoConnectionsOperation(resourceName string, isPscConfigExpected
 	}
 }
 
+func verifyPscNetorkAttachmentOperation(resourceName string, isPscConfigExpected bool, expectedPscEnabled bool, expectedNetworkAttachmentUri string ) func(*terraform.State) error {
```

Comment:
note: This is mostly redundant with import's builtin ImportStateVerify logic- which should do nearly the same thing- but no harm in keeping it around!